src/sdk/accountspace_fs.js

```diff
@@ -146,12 +141,10 @@ class AccountSpaceFS {
             // GAP - we do not have the user iam_path at this point (error message)
             this._check_if_requesting_account_is_root_account(action, requesting_account,
                 { username: params.username});
-            const account_config_path = this._get_account_config_path(params.username);
-            await this._check_if_account_config_file_exists(action, params.username, account_config_path);
-            const requested_account = await this._get_account_decrypted_data_optional(account_config_path, false);
+            await this._check_if_account_config_file_exists(action, params.username);
+            const requested_account = await this.configFS.get_account_by_name(params.username, true, true);
```

Comment:
it was without decryption:
`const requested_account = await this._get_account_decrypted_data_optional(account_config_path, false);`

the decryption happens only after we validate who is the requested account and his relation to the requested account.

Reply:
Why we need to decrypt it after this check?